boot/bootdata/hivedef.inf

```diff
@@ -2154,6 +2154,29 @@ HKCU,"AppEvents\EventLabels\WindowsLogoff","DispFileName",0x00000000,"@mmsys.cpl
 HKCU,"AppEvents\EventLabels\WindowsLogon","",0x00000000,"%EVENT_LABEL_27%"
 HKCU,"AppEvents\EventLabels\WindowsLogon","DispFileName",0x00000000,"@mmsys.cpl,-5853"
 
+; Sysdm.cpl Visual Effects
```

Comment:
Since these are "HKLM\Software...", these should be moved into hivesft.inf here; best place appears to be around line 458:
https://github.com/reactos/reactos/blob/b8400a2e1e46986a714d0c21830ec2e4b5c7da73/boot/bootdata/hivesft.inf#L457-L459